soc/intel/intel_adsp/Kconfig

```diff
@@ -93,11 +93,11 @@ config ADSP_INIT_HPSRAM
 	  Need to init HP SRAM.
 
 config ADSP_POWER_DOWN_HPSRAM
-	bool
+	bool "Switch off HP SRAM during power down"
```

Comment:
The addition of this help line makes this CONFIG_ "visible" and grants the user access to it, as described in the commit subject.

But then the rest of the changes look like (important?) bug fixes, described in the commit message.

So maybe I'm missing something but to me this really looks like this should be two different commits.

Reply:
@marc-hb well, can be either way - they're related, but can be separated too. This Kconfig change is small and simple enough IMHO so I merged them into a single commit